client/components/popover-menu/style.scss

```diff
@@ -12,7 +12,7 @@
 	border-radius: 0;
 	cursor: pointer;
 	display: block;
-	font-size: $font-body-small;
+	font-size: 0.875rem;
```

Comment:
Is this change intentional? It doesn't seem related to `border-radius`?

Reply:
Yeah, that variable was causing a build failure (I suspect because as part of the ETK it doesn't have access to the Calypso font size variables) so I updated it to hard-code the value. I should have added that to the description, thanks for catching it!

It remains to be seen why it wasn't an issue until now. 🤔